static/js/movie.js

```diff
@@ -0,0 +1,32 @@
+const ui = document.querySelector('.screenings');
+const path = window.location.pathname;
+
+async function getScreenings(){
+    const id = path.split('/').pop();
+    const res = await fetch(`/api/movies/${id}/screenings`);
+    const data = await res.json();
+    return data;
+}
+
+const data = getScreenings();
+
+data.then(data => {
+    let template = '<h2>Filmvisningar</h2>';
+    data.map(screening => {
+        console.log(screening.attributes)
+        const date = new Date(screening.attributes.start_time);
```

Comment:
console.log should be removed here and on site should time for screening be in format "20:00" not "20:00:00"